src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -115,6 +115,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ToolDepsJsonGeneratorProject>$(MSBuildThisFileDirectory)GenerateDeps\GenerateDeps.proj</ToolDepsJsonGeneratorProject>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
We want to stop generating this file for 6.0 onwards. Previous versions should keep producing the file.

Reply:
Whoops, thanks!